test/e2e/operator_test.go

```diff
@@ -24,6 +24,7 @@ import (
 	"testing"
 	"time"
 
+	"github.com/stretchr/testify/assert"
```

Comment:
This import needs to be reordered alphabetically.

Reply:
It's already alphabetical (and gofmt is happy with it).  Do you mean it needs to be regrouped?  I shied away from that given that the current grouping is inconsistent and we already have plans to fix it throughout our codebase, but I can fix up this file now if you like.  

Would you like me to regroup as follows?
```go
import (
	"bufio"
	"bytes"
	"context"
	"crypto/tls"
	"crypto/x509"
	"encoding/json"
	"errors"
	"fmt"
	"io"
	"io/ioutil"
	"net"
	"net/http"
	"os"
	"reflect"
	"sort"
	"strconv"
	"strings"
	"testing"
	"time"

	"github.com/aws/aws-sdk-go/aws/endpoints"
	"github.com/go-logr/logr"
	configv1 "github.com/openshift/api/config/v1"
	operatorv1 "github.com/openshift/api/operator/v1"
	iov1 "github.com/openshift/api/operatoringress/v1"
	routev1 "github.com/openshift/api/route/v1"
	"github.com/stretchr/testify/assert"
	"gopkg.in/yaml.v2"
	appsv1 "k8s.io/api/apps/v1"
	corev1 "k8s.io/api/core/v1"
	networkingv1 "k8s.io/api/networking/v1"
	policyv1 "k8s.io/api/policy/v1"
	apierrors "k8s.io/apimachinery/pkg/api/errors"
	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
	"k8s.io/apimachinery/pkg/runtime"
	"k8s.io/apimachinery/pkg/runtime/schema"
	"k8s.io/apimachinery/pkg/types"
	"k8s.io/apimachinery/pkg/util/intstr"
	"k8s.io/apimachinery/pkg/util/sets"
	"k8s.io/apimachinery/pkg/util/wait"
	"k8s.io/apiserver/pkg/storage/names"
	discocache "k8s.io/client-go/discovery/cached"
	"k8s.io/client-go/dynamic"
	"k8s.io/client-go/kubernetes"
	"k8s.io/client-go/restmapper"
	"k8s.io/client-go/scale"
	"k8s.io/utils/pointer"
	"sigs.k8s.io/controller-runtime/pkg/client"
	"sigs.k8s.io/controller-runtime/pkg/client/config"
	ctrlruntimelog "sigs.k8s.io/controller-runtime/pkg/log"

	"github.com/openshift/cluster-ingress-operator/pkg/manifests"
	operatorclient "github.com/openshift/cluster-ingress-operator/pkg/operator/client"
	"github.com/openshift/cluster-ingress-operator/pkg/operator/controller"
	operatorcontroller "github.com/openshift/cluster-ingress-operator/pkg/operator/controller"
	ingresscontroller "github.com/openshift/cluster-ingress-operator/pkg/operator/controller/ingress"
)
```

Reply:
I hadn't noticed how unordered the whole list was, only that the new import did not appear to be in the right place.  

I would be happy if you just added the new import after:
"github.com/go-logr/logr"

But if you don't want to do that without fixing all the others, I get it.

Reply:
The regroup isn't grouping all the github.com/openshift together, so definitely not that.

Reply:
Do you want me to group `"github.com/go-logr/logr"` with `"gopkg.in/yaml.v2"`, or just move `"github.com/stretchr/testify/assert"`?

Reply:
https://github.com/openshift/cluster-ingress-operator/compare/cfba4d9ad778d01a6ab874682ad0c2d95f81eab0..7599a46d8ecfee4b398bd5f228aceb1040c2c165 moves `"github.com/stretchr/testify/assert"` to be with `"github.com/go-logr/logr"`.